app/views/fe/answer_pages/_element.html.erb

```diff
@@ -7,7 +7,8 @@
 <% end -%>
 
 <% begin %>
-  <%= render :partial => 'fe/questions/' + element.ptemplate, :object => element, :locals => {:col => nil} %>
+  <% col ||= nil %>
+  <%= render :partial => 'fe/questions/' + element.ptemplate, :object => element, :locals => {:col => col} %>
 <% rescue => ex %>
```

Comment:
While you're in there with this code anyway... might as well clean this up? obviously rescuing ex then raising it doesn't do much.